src/python/WMCore/WMSpec/StdSpecs/StdBase.py

```diff
@@ -154,12 +154,13 @@ def skimToDataTier(cmsswVersion, skim):
         command += "cd %s\n" % scramBaseDirs[0]
         command += "eval `scramv1 runtime -sh`\n"
 
-        command += """python -c 'from Configuration.StandardSequences.Skims_cff import getSkimDataTier\n"""
+        command += """python3 -c 'from Configuration.StandardSequences.Skims_cff import getSkimDataTier\n"""
         command += """dataTier = getSkimDataTier("%s")\n""" % skim
-        command += """if dataTier:\n\tprint dataTier.value()'"""
+        command += """if dataTier:\n\tprint (dataTier.value())'"""
 
+        logging.info("command= %s" % command)
         p = subprocess.Popen(command, stdout=subprocess.PIPE, shell=True)
-        dataTier = p.communicate()[0].strip()
+        dataTier = p.communicate()[0].decode('utf-8').strip()
```

Comment:
Same comment as above, regarding the adoption of `decodeBytesToUnicodeConditional`